pages*: improve page descriptions (#12558)
tldr-pages/tldr: pages/linux/cockpit-desktop.md

# cockpit-desktop

> > Securely access Cockpit pages in an already running session.

Reviewer: ```suggestion
> Securely access Cockpit pages in a running session.
```
